app/src/main/java/ch/admin/bag/dp3t/networking/models/StatsResponseModel.java

```diff
@@ -9,18 +9,19 @@
  */
 package ch.admin.bag.dp3t.networking.models;
 
-import java.text.DecimalFormat;
 import java.util.Date;
 import java.util.List;
 
 import ch.admin.bag.dp3t.util.DateUtils;
 
 public class StatsResponseModel {
 
-	private static double ONE_MILLION = 1000000;
-
 	private String lastUpdated;
 	private int totalActiveUsers;
```

Comment:
For consistency and just in case we should also handle the null case for totalActiveUsers.

Reply:
Makes sense. Fixed that